apps/Gruntfile.js

```diff
@@ -429,7 +429,8 @@ testsContext.keys().forEach(testsContext);
     'raceInterstitial':             './src/sites/studio/pages/raceInterstitial.js',
     'layouts/_terms_interstitial':  './src/sites/studio/pages/layouts/_terms_interstitial.js',
     'maker/setup':                  './src/sites/studio/pages/maker/setup.js',
-    'scriptOverview':               './src/sites/studio/pages/scriptOverview.js'
+    'scriptOverview':               './src/sites/studio/pages/scriptOverview.js',
+    'home/new_teacher_homepage':         './src/sites/studio/pages/home/new_teacher_homepage.js'
```

Comment:
Also does it make sense to create `pages/home/` or is it okay to just put in `pages/'?